bin/test-db

```diff
@@ -71,12 +86,12 @@ def stop_container(name):
     print("Process stopped successfully")
 
 def connect_to_db(name):
-    CONNECT_COMMAND = "docker run -it --rm mongo mongo --host {} test -u {} -p {} --authenticationDatabase admin"
-
+    CONNECT_COMMAND = "docker run -it --rm {} --host {} test -u {} -p {} --authenticationDatabase admin"
```

Comment:
This change
```diff
  CONNECT_COMMAND = "docker run -it \
-                           --rm mongo mongo \
                            --host
  CONNECT_COMMAND = "docker run -it \
+                           --rm {} \
                            r--host
```
is tricky outside of the context of a code review.

With the diff visible in a code review, it's easy to see that the new version is going to format in the docker image to run and the command to run as one string. But I think it's easy to see how that can get confusing super fast.

`conn_option` should not be the two pieces combined. We should find another way to update this `CONNECT_COMMAND`

Reply:
Split out into two fields 'image' and 'shell'.